bims/static/js/views/map_control_panel.js

```diff
@@ -1,15 +1,21 @@
-define(['backbone', 'underscore', 'jquery', 'ol', 'views/search'], function (Backbone, _, $, ol, SearchView) {
+define(
+    ['backbone', 'underscore', 'jquery', 'ol', 'views/search', 'views/locate'],
```

Comment:
can you remove one view locate between from map_control_panel or olmap?

Reply:
Done, thanks for the review @meomancer